azext_iot/central/commands_enrollment_group.py

```diff
@@ -0,0 +1,300 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# Dev note - think of this as a controller
+
+from typing import List
+from azext_iot.constants import CENTRAL_ENDPOINT
+from azext_iot.central.providers import CentralEnrollmentGroupProvider
+from azext_iot.common import utility
+from azext_iot.central.common import API_VERSION
+from azext_iot.central.models.ga_2022_07_31 import EnrollmentGroupGa
+
+
+def get_enrollment_group(
+    cmd,
+    app_id: str,
+    group_id: str,
+    entry: str = None,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=API_VERSION,
+) -> EnrollmentGroupGa:
+    provider = CentralEnrollmentGroupProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    response = provider.get_enrollment_group(
+        group_id=group_id,
+        central_dns_suffix=central_dns_suffix,
+    )
+
+    if entry is not None:
+        response["x509"] = get_x509(
+            cmd=cmd,
+            app_id=app_id,
+            group_id=group_id,
+            entry=entry,
+            token=token,
+            central_dns_suffix=central_dns_suffix,
+            api_version=api_version,
+        )
+
+    return response
+
+
+def list_enrollment_groups(
+    cmd,
+    app_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=API_VERSION,
+) -> List[EnrollmentGroupGa]:
+    provider = CentralEnrollmentGroupProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    return provider.list_enrollment_groups(central_dns_suffix=central_dns_suffix)
+
+
+def create_enrollment_group(
+    cmd,
+    app_id: str,
+    attestation: str,
+    display_name: str,
+    type: str,
+    group_id: str,
+    enabled: bool = False,
```

Comment:
by default, you create a disabled enrollment group? DPS has enabled by default (see `az iot dps enrollment-group create`, specifically `--provisioning-status`)

Reply:
Oh I didn't notice this. I'll change default to True.